recipes/localhgt/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Ensure we run successfully using either conda-forge or defaults ncurses
+# (unlike other platforms, the latter does not automatically pull in libtinfo)
```

Comment:
The comment above is not very clear to me. How ncurses (a TUI library) is related here ?

Reply:
Hi, thanks for the review. The comment is nonsense. So, I delete them.